app/models/market_model.rb

```diff
@@ -0,0 +1,17 @@
+class MarketModel < ActiveRecord::Base
+  include Privacy
+
+  FOUNDATIONS = ["kWh", "kW_connection", "kW_max", "kW_contracted", "kW_flex"]
```

Comment:
> I actually think that 'kWh' and 'kW_connection' are going to be the most important 

Fine with me. :+1: kWh is probably the simplest of the two, so I think I'll start there.